### DISCUSSION TITLE
Phishing report for several vercel.app sites targeting PancakeSwap

### DISCUSSION THREAD

**Discussion by OctavioSardi** [vercel/community#6934] [Help]
### Summary

Hi!

I'm approaching you to follow up on a few abuse cases that have been properly reported to abuse@vercel.com, but are yet to be attended. I've confirmed with PancakeSwap and none of this sites are affiliated with them. Plus all of this sites use their name and branding to request wallet connection, endangering user's crypto assets.

- **Vercel Case Number:** 00231298 - **URL:** hxxps[:]// pancakeairdropsclicks03[.]vercel[.]app
- **Vercel Case Number:** 00231296 - **URL:** hxxps[:]// pancakeswapdropsclaim02[.]vercel[.]app
- **Vercel Case Number:** 00225933 - **URL:** hxxps[:]//base-swap-frontend-klt095har-truest422[.]vercel[.]app
- **Vercel Case Number:** 00225935 - **URL:** hxxps[:]//pancakeswap-frontend-git-fix-refactors-and-bugs-dopex-io[.]vercel[.]app

Some of this cases are over a month old, please give us a hand so they can be removed swiftly.

Many thanks for your help!

### Steps to Reproduce

Enter the to the reported URL.

**Comment by OctavioSardi** [comment]
cc @amyegan

**Reply by amyegan** [reply]
Thanks for letting us know about these pages you believe to be problematic. You already did the right thing by submitting your reports to [abuse@vercel.com](mailto:abuse@vercel.com). You will receive a response when that team has completed their review of your report. Please follow up with your open cases if you have any questions or additional information to share with them.

If you have a copyright or DMCA report, please email [dmca@vercel.com](mailto:dmca@vercel.com). When doing this, please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) or [trademark policy](https://vercel.com/legal/trademark-policy).
